go-carbon.conf.example

```diff
@@ -426,6 +426,12 @@ stats-percentiles = [99, 98, 95, 75, 50]
 # path = "/metrics/list_query/"
 # max-inflight-requests = 3
 
+# carbonserver.grpc is the configuration for listening for grpc clients.
+# [carbonserver.grpc]
+# enabled = true
+# listen = ":7004"
```

Comment:
oh, can you also add that only `/render` is currently supported, and copy this change to [the readme configuration section](https://github.com/go-graphite/go-carbon#configuration) as well? tx.

Reply:
OK. But why are we maintaining another configuration example inside README? We can just put a link to the example configuration instead, can't we?

Reply:
maybe. for this pr, let's stick to the duplication. a little bit of copy doesn't hurt. ;)

I'm doing it mainly because they were there, so I just keep them as up-to-date and matching as possible.

In readme it improves visibility of new features, in example conf file it makes it easy for people to just use it. but it's a random opinion.